lib/graphiti/delegates/pagination.rb

```diff
@@ -6,7 +6,7 @@ def initialize(proxy)
       end
 
       def links?
-        @proxy.query.pagination_links?
+        @proxy.data.present? && @proxy.query.pagination_links?
```

Comment:
adding an extra check of `@proxy.data.present?` here. Do we really need to render pagination links if the records on requested endpoint are empty?